core/jazz_ui/src/app/core/services/admin-utils.service.ts

```diff
@@ -1,12 +1,11 @@
-import {Injectable} from '@angular/core';
-import {Http} from "@angular/http";
-import {Observable} from "rxjs/Observable";
-import {RequestService} from "./request.service";
+import { Injectable } from '@angular/core';
+import { Http } from "@angular/http";
+import { Observable } from "rxjs/Observable";
+import { RequestService } from "./request.service";
```

Comment:
can we be consistent, probably without spaces like {Observable}

Reply:
Done the changes.